### ISSUE TITLE
config_full_document_before not working in EventTriggers

### ISSUE THREAD

**Issue by jon-king-mindbodyonline** [mongodb/terraform-provider-mongodbatlas#616]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.0.11
on darwin_amd64
+ provider registry.terraform.io/mongodb/mongodbatlas v1.1.1
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_event_trigger" "test" {
  project_id = "123456"
  app_id = "123456"
  name = "test-trigger"
  type = "DATABASE"
  disabled = false
  config_operation_types = ["INSERT", "UPDATE"]
  config_database = "dbname"
  config_collection = "cname"
  config_service_id = "123456"
  config_match = "{\"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
  config_project = "{\"operationType\": 1, \"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
  config_full_document = true
  config_full_document_before = true
  event_processors {
    aws_eventbridge {
      config_account_id = "123456"
      config_region = "us-west-1"
    }
  }
}
```

### Steps to Reproduce
1. `terraform init`
2. `terraform apply`
3. Check the Atlas UI for the Trigger
4. Also check the Realm API directly
```sh
> BEARERTOKEN=$(curl -s --request POST  --header 'Content-Type: application/json' \
  --header 'Accept: application/json' \
  --data '{"username": "<public-key>", "apiKey": "<private-key>"}' \
  https://realm.mongodb.com/api/admin/v3.0/auth/providers/mongodb-cloud/login | jq -r '.access_token')
> curl --request GET \
--header 'Content-Type: application/json' \
--header "Authorization: Bearer ${BEARERTOKEN}" \
https://realm.mongodb.com/api/admin/v3.0/groups/<project_id>/apps/<application_id>/triggers
```

### Expected Behavior
That the trigger be created in Realm with "Document Preimage" enabled (`FullDocumentBeforeChange` in the API and Go library)

### Actual Behavior
Trigger is successfully created but with Document Preimage false, confirmed in Atlas UI and API.
There's a good chance, based on trace output, this is a Realm API issue:

### Debug Output 
[trace.log](https://github.com/mongodb/terraform-provider-mongodbatlas/files/7573620/trace.log)

**Comment by themantissa** [comment]
@jon-king-mindbodyonline thank you for the complete report.  I've created an internal ticket to have our developers take a look.   We'll get back to you as soon as we have more information.   ticket INTMDB-281

**Comment by themantissa** [comment]
@jon-king-mindbodyonline  we are about to release the fix in version 1.2.  The pre-release is completed if you'd like to test first.  Thank you.

**Comment by themantissa** [comment]
Fix in 1.2
